src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/buildAllAgentListContext.hpp

```diff
@@ -67,9 +68,14 @@ class TBuildAllAgentListContext final : public AbstractHandler<std::shared_ptr<T
             wdbSocketWrapper.query(WazuhDBQueryBuilder::builder().global().selectAll().fromTable("agent").build(),
                                    response);
         }
-        catch (const std::exception&)
+        catch (const SocketDbWrapperException& e)
         {
-            logDebug2(WM_VULNSCAN_LOGTAG, "Error executing query to fetch global agent data");
+            logWarn(WM_VULNSCAN_LOGTAG, "Failed to executing query to fetch global agent data. %s", e.what());
+            throw WdbDataException("Error executing query to fetch global agent data", "all");
+        }
+        catch (const std::exception& e)
+        {
+            logError(WM_VULNSCAN_LOGTAG, "Error executing query to fetch global agent data. %s", e.what());
             throw WdbDataException("Error executing query to fetch global agent data", "all");
         }
```

Comment:
Remove the warn and error logging from this, we are going to log on the wrapper

Add a debug log if necessary